<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>377</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>350</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BTSearcher</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/icon/app_icon.png</normaloff>:/res/icon/app_icon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	font: 9pt &quot;Montserrat&quot;;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(253, 238, 255, 150), stop:1 rgba(229, 241, 255, 150));
}

QPushButton {
	font: 700 8pt &quot;Montserrat&quot;;
	color: rgb(95, 99, 104);
        background-color: rgb(255, 255, 255);
	border-style: outset;
	border-width: 5px;
	border-radius: 10px;
	border-color: rgba(255, 225, 255, 30);
	padding: 10px 5px 10px 5px;
}
QPushButton:hover {
    background-color: rgb(239, 239, 239);
}
QPushButton:pressed {
    background-color: rgb(228, 227, 231);
}
</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>741</width>
      <height>352</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QTextEdit" name="consoleOutput">
       <property name="whatsThis">
        <string>console</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QTextEdit {
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 5px 5px 5px 5px;
font: 8pt &quot;Montserrat&quot;;
}

QScrollBar:vertical {}
</string>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="documentTitle">
        <string comment="console"/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Montserrat'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; -qt-user-state:131073; font-family:'Segoe UI'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 3px;
padding: 3px 5px 3px 5px;
</string>
         </property>
         <property name="text">
          <string>Раскрыть</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 3px;</string>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerItem</enum>
         </property>
         <property name="allColumnsShowFocus">
          <bool>false</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Дерево</string>
          </property>
          <property name="background">
           <color>
            <red>179</red>
            <green>179</green>
            <blue>179</blue>
           </color>
          </property>
          <property name="foreground">
           <brush brushstyle="NoBrush">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>0</blue>
            </color>
           </brush>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="buttonLayout">
       <item>
        <widget class="QPushButton" name="searchInThree">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Найти элемент</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="insertInThree">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Вставить элемент</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteInThree">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Удалить элемент</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="inorder">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Inorder обход</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="preorder">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Preorder обход</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="postorder">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Postorder обход</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>754</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QMenuBar {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(253, 238, 255, 150), stop:1 rgba(229, 241, 255, 100));

}

QMenu {
	border: 1px solid ;
	border-color: rgb(241, 241, 237);
	background-color: rgb(255, 255, 255);
}

QMenu::item::selected {
	background-color: rgba(232, 232, 232, 200);
	color: rgb(67, 67, 67)
}</string>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="new_file"/>
    <addaction name="openFile"/>
    <addaction name="saveAs"/>
   </widget>
   <widget class="QMenu" name="info">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Инфо</string>
    </property>
    <addaction name="about"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="info"/>
  </widget>
  <action name="openFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icon/open.png</normaloff>:/res/icon/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="saveAs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icon/saveAs.png</normaloff>:/res/icon/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="about">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icon/info.png</normaloff>:/res/icon/info.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="new_file">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/icon/new_file_.png</normaloff>:/res/icon/new_file_.png</iconset>
   </property>
   <property name="text">
    <string>Новый файл</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
